sections/Pools/PoolDurationEnded/PoolDurationEndedBox/PoolDurationEndedBox.tsx

```diff
@@ -0,0 +1,107 @@
+import { FC, useMemo, useEffect, useState } from 'react';
+
+import Connector from 'containers/Connector';
+import TransactionData from 'containers/TransactionData';
+
+import ConfirmTransactionModal from 'components/ConfirmTransactionModal';
+
+import {
+	Container,
+	ErrorNote,
+	ContentContainer,
+	ActionBoxInputLabel,
+	InputContainer,
+	ActionBoxInput,
+	ActionBoxMax,
+	ActionButton,
+} from '../../../shared/common';
+
+import { TransactionStatus, TransactionType } from 'constants/transactions';
+import { GasLimitEstimate } from 'constants/networks';
+
+interface PoolDurationEndedBoxProps {
+	onSubmit: () => void;
+	inputValue: number | string;
+	setInputValue: (value: number | string) => void;
+	setIsMaxValue: (value: boolean) => void;
+	userPoolBalance: number;
+	gasLimitEstimate: GasLimitEstimate;
+	purchaseTokenSymbol: string;
+}
+
+const PoolDurationEndedBox: FC<PoolDurationEndedBoxProps> = ({
+	onSubmit,
+	userPoolBalance,
+	purchaseTokenSymbol,
+	gasLimitEstimate,
+	inputValue,
+	setInputValue,
+	setIsMaxValue,
+}: any) => {
+	const { walletAddress } = Connector.useContainer();
+	const { setGasPrice } = TransactionData.useContainer();
+
+	const [showTxModal, setShowTxModal] = useState(false);
+	const isEmptyInput = inputValue === '' || inputValue === 0;
+	const isMaxBalanceExceeded = Number(userPoolBalance ?? 0) < Number(inputValue ?? 0);
+
+	const isDisabled: boolean = useMemo(() => {
+		return !walletAddress || isMaxBalanceExceeded || isEmptyInput;
+	}, [walletAddress, isMaxBalanceExceeded, isEmptyInput]);
+
+	return (
+		<Container>
+			<ContentContainer>
+				<ActionBoxInputLabel>{`Balance ${userPoolBalance} Pool Tokens`}</ActionBoxInputLabel>
+				<InputContainer>
+					<ActionBoxInput
+						type="number"
+						placeholder="0"
+						value={inputValue}
+						onChange={(e) => {
+							const value = !!e.target.value.length ? parseFloat(e.target.value) : '';
```

Comment:
can we just do const value = !!e.target.value.length ? parseFloat(e.target.value) : 0 and remove all the empty string logic and inputValue will always just be a number then?